### DISCUSSION TITLE
Can I deploy saleor-platform as production ?

### DISCUSSION THREAD

**Discussion by phearakthefoundry** [saleor/saleor#6453] [Deployment]
Please, bear in mind that Saleor-Platform repository has only been designed for local development. It's mean that Saleor-Platform it designed for testing only or we can take it to customize and deploy as production ?

**Comment by karolkielecki** [comment]
@phearakthefoundry Hey! Dockerfile inside Saleor is ready for production. You just need to build and deploy the image - https://docs.saleor.io/docs/developer/running-saleor/docker.
Please note that the compose file (on platform repository level) is strictly for development, as it's stated in the readme section.

**Reply by phearakthefoundry** [reply]
it's not full step.
